plugins/source/aws/resources/services/ecs/cluster_services.go

```diff
@@ -26,6 +26,14 @@ func ClusterServices() *schema.Table {
 					PrimaryKey: true,
 				},
 			},
+			{
```

Comment:
instead of this, use `transformers.WithPrimaryKeys("ClusterArn")`

Reply:
Fixed